apiserver/facades/client/applicationoffers/mock_test.go

```diff
@@ -492,8 +493,8 @@ func (m *mockState) APIHostPorts() ([][]network.HostPort, error) {
 	}, nil
 }
 
-func (m *mockState) CACert() string {
-	return testing.CACert
+func (m *mockState) ControllerConfig() (controller.Config, error) {
```

Comment:
Is this used anywhere?

Reply:
It was used for the backend of the APIAddresser. I've updated the PR with a commit to remove CACert from that interface.